src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -101,6 +75,20 @@ void createStationWithDuplicateName() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        지하철역_생성_요청("강남역");
+        지하철역_생성_요청("잠실역");
+
+        // when
+        ExtractableResponse<Response> 조회_응답 = 지하철역_목록_조회();
+
+        // then
+        지하철역_조회_성공_확인(조회_응답);
+
+        // then
+        List<String> 지하철역_이름_목록 = 조회_응답.jsonPath().getList("name", String.class);
+        지하철역_포함_확인(지하철역_이름_목록, "강남역");
```

Comment:
```suggestion
        지하철역_포함_확인(조회_응답, "강남역");
```
위와 같이 89 라인도 숨길수 있지 않을까요.

물론 89 라인이 여러번 실행 되는 단점이 생기지만 가독성 측면에서는 더 좋아보이는데 
어떻게 생각하시나요?

Reply:
확실히 가독성 측면에서는 훨씬 좋아보이네요!
안그래도 조금 고민했던 부분인데 좋은 방법인 것 같아요.
수정해서 올리겠습니다.